src/main/java/org/sciborgs1155/robot/shooter/feeder/FeederIO.java

```diff
@@ -0,0 +1,5 @@
+package org.sciborgs1155.robot.shooter.feeder;
+
+public interface FeederIO extends AutoCloseable {
```

Comment:
Since `FeederIO` `RealFeeder` and `SimFeeder` feature so little code, it might be cleaner to put them all inside a single `FeederIO.java` file